wagtail_streamfield_migration_toolkit/test/tests/test_streamchangedetector.py

```diff
@@ -0,0 +1,112 @@
+from unittest import mock
+from django.test import TestCase
+from wagtail.blocks import CharBlock, StreamBlock
+
+from wagtail_streamfield_migration_toolkit.autodetect.streamchangedetector import (
+    StreamDefChangeDetector,
+)
+from wagtail_streamfield_migration_toolkit.operations import (
+    RenameStreamChildrenOperation,
+    RemoveStreamChildrenOperation,
+)
+from ..models import (
+    BaseStreamBlock,
+    SimpleStreamBlock,
+)
+
+
+class SimpleStreamWithRemovedChild(StreamBlock):
+    # removed char 1
+    char2 = CharBlock()
+
+
+class SimpleStreamWithRenamedChild(StreamBlock):
+    renamed_char1 = CharBlock()
+    char2 = CharBlock()
+
+
+class StreamChangeDetectorTests(TestCase):
+    def assertRemoveOperationEqual(
+        self, operation_and_block_path1, operation_and_block_path2
+    ):
+        self.assertEqual(operation_and_block_path1[1], operation_and_block_path2[1])
+        self.assertEqual(
+            operation_and_block_path1[0].__class__,
+            operation_and_block_path2[0].__class__,
+        )
+        self.assertEqual(
+            operation_and_block_path1[0].name, operation_and_block_path2[0].name
+        )
+
+    def assertRenameOperationEqual(
+        self, operation_and_block_path1, operation_and_block_path2
+    ):
+        self.assertEqual(operation_and_block_path1[1], operation_and_block_path2[1])
+        self.assertEqual(
+            operation_and_block_path1[0].__class__,
+            operation_and_block_path2[0].__class__,
+        )
+        self.assertEqual(
+            operation_and_block_path1[0].old_name, operation_and_block_path2[0].old_name
+        )
+        self.assertEqual(
+            operation_and_block_path1[0].new_name, operation_and_block_path2[0].new_name
+        )
+
+    def test_same(self):
+        """Test the same block definition"""
```

Comment:
Test that the same block definition... does what? Maybe worth explicitly saying "produces no operations when compared", or similar